packages/cli/src/databases/migrations/mysqldb/1690000000031-FixExecutionDataType.ts

```diff
@@ -0,0 +1,9 @@
+import type { MigrationContext, IrreversibleMigration } from '@db/types';
+
+export class FixExecutionDataType1690000000030 implements IrreversibleMigration {
+	async up({ queryRunner, tablePrefix }: MigrationContext) {
+		await queryRunner.query(
+			'ALTER TABLE `' + tablePrefix + 'execution_data` MODIFY COLUMN `data` MEDIUMTEXT',
```

Comment:
This ensures that schema is consistent for anyone who managed to upgrade to `0.234.x` or `0.235.x`.

Reply:
Can we add the reason as an actual comment? More generally, maybe we should consider starting to add short text summaries to migrations.

Reply:
done.